### DISCUSSION TITLE
Taro 插件：让 Taro 小程序可以无缝兼容 Tailwind/Windi CSS 以及其中的 JIT 功能

### DISCUSSION THREAD

**Discussion by ItsRyanWu** [NervJS/taro#11815] [生态]
开源项目地址：[mini-program-tailwind](https://github.com/dcasia/mini-program-tailwind)，其中包含“一键安装”的 Taro 插件。
 
由于小程序本身不支持由 Tailwind/Windi CSS 产生的选择器名称中包含的一些特殊转义字符（如 `\[` `\!` `\.` 等），这使得你在开发小程序时无法使用一些本该在开发 Web 应用时使用的得心应手的灵活语法与 JIT/Value auto-infer 功能，如 `w-[30px]` `translate-x-1/2` `!bg-[#ff0000]` 等。这无疑对我们的开发效率与心智负担带来了不小的影响。

为了突破这一限制，我们开发了这一款插件来帮助你在使用 Tailwind/Windi CSS 开发小程序时仍然保持着与开发 Web 应用高度一致的开发体验，你不再需要关注因为哪些字符串不被支持而不得不换一种写法的问题，而是继续**按照 Tailwind/Windi CSS 的官方语法继续编写你的小程序样式**，背后的兼容工作则由这款插件静默处理。

> <details>
>     <summary>进一步了解本插件的工作原理</summary>
>     
> [让你的小程序用上原汁原味的 Tailwind/Windi CSS](https://juejin.cn/post/7093809282272985119/)
> </details>

希望感兴趣的各位可以使用看看，相比于目前社区推荐的 Tailwind 插件，本插件可以兼容 Tailwind/Windi CSS 的 JIT 功能。有新需求或者 bug 欢迎向 [mini-program-tailwind](https://github.com/dcasia/mini-program-tailwind) 提交 issue，我会及时处理。

**Comment by airqj** [comment]
赞👍🏻

**Reply by ItsRyanWu** [reply]
欢迎使用并提意见😌
